modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/form-builder/_share_url_popover.scss

```diff
@@ -1,14 +1,6 @@
 $c: '.ddm-share-url-popover';
```

Comment:
@brunofariasdeo, from what I remember, we realized that this class wasn't used too, right? Can you please double check? If this class isn't being used too, please remove it.

Reply:
Hey @brunofariasdeo, have you had the chance to check my comment above? Thanks.

Reply:
Hi @natocesarrego, as discussed, I've deleted the css file once ddm-share-url-popover is not being used.